ci/test/00_setup_env_s390x.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export HOST=s390x-unknown-linux-gnu
+export DOCKER_NAME_TAG=s390x/ubuntu:18.04
```

Comment:
```suggestion
```

Is this needed?

Reply:
I'll check, in the PowerPC one Docker failed to download the right image for the platform (and instead downloaded amd64)